File: Backend/Models/Store.cs
namespace Backend.Models
{
    public class Store
    {
        public int id { get; set; }
        public string name { get; set; }
        public string email { get; set; }
        public string province { get; set; }
        public string city { get; set; }
        public string district { get; set; }
        public int? managerId { get; set; }
        public string? managerName { get; set; }
        public string? managerLastName1 { get; set; }
        public string? managerLastName2 { get; set; }
        public int? storeId { get; set; }
        public string? storeType { get; set; }
        public List<string>? phoneNumbers { get; set; }
    }
}

File: Frontend/src/app/Interfaces/Store.ts
interface Store {
  id: number
  name: string
  email: string
  province: string
  city: string
  district: string
  managerID: number
  storeTypeID: number
  storeTypeName: string
  phoneNumbers: string[]
}

interface StoreManager {
  id: number
  username: string
  name: string
  lastName1: string
  lastName2: string
  email: string
  province: string
  city: string
  district: string
  phoneNumbers: string[]
}

interface StoreData {
  store: Store
  manager: StoreManager
}

export {
  Store,
  StoreManager,
  StoreData
}